islandora_basic_collection.module

```diff
@@ -1092,6 +1103,7 @@ function islandora_basic_collection_block_save($delta = '', $edit = array()) {
     variable_set('islandora_basic_collection_object_count_listing_content_models_to_restrict', $edit['content_models']);
     variable_set('islandora_basic_collection_object_count_listing_phrase', $edit['islandora_basic_collection_title_phrase']);
     variable_set('islandora_basic_collection_object_count_listing_placeholder', $edit['islandora_basic_collection_title_placeholder']);
+    variable_set('islandora_basic_collection_count_method', $edit['islandora_basic_collection_count_method']);
```

Comment:
islandora_basic_collection_count_method needs to go to .install file too in case of uninstalling (I know..not much people are going to uninstall)

Reply:
Ack, silly thing to forget, thanks.